### ISSUE TITLE
yank rdf-open_annotation gem?

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/rdf-open_annotation#5]
http://help.rubygems.org/kb/gemcutter/removing-a-published-rubygem ?

version 0.1.0 with deprecation warnings released 2015-04-28

**Comment by dazza-codes** [comment]
Yes, yank it so the namespace becomes available for something else that is not deprecated.
